lib/core/tests/text.rs

```diff
@@ -1,18 +1,17 @@
-//! Test suite for the Web and headless browsers.
```

Comment:
If this file is commented out maybe lets remove it till we dont have new tests here?